### ISSUE TITLE
Identify <tr> and <td> pertaining to class `Sub-Category` in Country page

### ISSUE THREAD

**Issue by rnllv** [DeepBlueCLtd/LegacyMan#45]
Below is the `class` table in Narnia1.html.

Are we missing some other patterns that will help us efficiently identify `Sub-Category` from this table?

Below approaches aren't foolproof.

* Any < tr > tag with `colspan` attribute set.
This will not work as `Link is to French unit_d` has also `colspan` attribute set.
* If `colspan` approach is considered, we'll need to hardcode `Link is to French unit_d` to prevent accessing that row

As previously mentioned, hardcoding `Legacy` and `Composite` in the parser code will not work against real data.
```
<table border="1">
                <tbody><tr>
                    <td>Overview of systems in Narnia</td>
                </tr>
                <tr>
                    <td>Class</td>
                    <td>Designator</td>
                    <td>Power</td>
                    <td>Shaft x Blade</td>
                    <td>BHP</td>
                    <td>Av Temp</td>
                    <td>Reduction Ratio</td>
                </tr>
                <tr colspan="7"><td>Composite</td></tr>
                <tr>
                    <td>Unit_a</td>
                    <td>PB</td>
                    <td>2 * V8 Diesel</td>
                    <td>3 x 3</td>
                    <td>34 Hz</td>
                    <td>Unk</td>
                    <td>Unk</td>
                </tr>
                <tr>
                    <td>Unit_b</td>
                    <td>PBF</td>
                    <td>2 * V6 Diesel</td>
                    <td>2 * 4 (FPP)</td>
                    <td>16.6</td>
                    <td>Est 1000</td>
                    <td>BWW</td>
                </tr>
                <tr colspan="7"><td>Legacy</td></tr>
                <tr>
                    <td>Unit_c</td>
                    <td>PBA</td>
                    <td>5 * V8 Diesel</td>
                    <td>1 x 3</td>
                    <td>300 Hz</td>
                    <td>JR</td>
                    <td>Unk</td>
                </tr>
                <tr>
                    <td><span>*</span><a href="../France_Legacy/unit_d.html">Unit_d</a></td>
                    <td>PBF</td>
                    <td>2 * V6 Diesel</td>
                    <td>2 * 4 (FPP)</td>
                    <td>16.6</td>
                    <td>Est 1000</td>
                    <td>BWW</td>
                </tr>
                <tr>
                    <td colspan="7">Link is to French unit_d</td>
                </tr>
            </tbody></table>
```

**Comment by IanMayo** [comment]
I'll have to examine the data.  But, we're fortunate in the above example.  No classes follow the colspan, so we wouldn't use the category type. We _could_ test that the colspan is category rather than description by testing that it is 3 words or less.

**Comment by rnllv** [comment]
Please let me know your thoughts on #46

**Comment by IanMayo** [comment]
_Maybe_ there is merit in my inspecting the list of `Sub-Category` values.  I can look for near-duplicates (and correct them, and for entries that should not be in there (to produce an exclusion list).

**Update:** I fixed about 20 instances of leading spaces, swapping `&` for `&amp;` or `/`.  There are still some instances of `Entity` vs `Entities`.  I'll work with author to see if we can settle on plural instance.  There remain lots of `This is a link to blah, blah`.  I've realised we can skip those as sub-category types since sub-category types have a filled background shade.
